acme-cluster/acmeRepo/public_html/content/remote/login/step1.html

```diff
@@ -0,0 +1,69 @@
+<!--
```

Comment:
In Acme Cluster we should not setup Filesystem Repository. 
Filesystem repository is used in Acme Stack only.

We should have exchange all  `/content/remote/*` to `/content/*`.

Reply:
We agreed that the sample content must be refactored in a separate issue.